bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -173,19 +186,6 @@ public void thingUpdated(Thing thing) {
         initialize();
     }
 
-    public void configurationChanged(Map<String, String> changedParameters) throws Exception {
```

Comment:
This method came accidentally in by another PR. It should never had made it into ESH, sorry for that. But I guess no one used it.